configure

```diff
@@ -583,6 +618,12 @@ while [[ ! -z $1 ]] ; do
 #      HDF5LIB="-I$VALUE/include $VALUE/lib/libhdf5_hl.a $VALUE/lib/libhdf5.a -lm"
 #      echo "Using HDF5LIB: $HDF5LIB"
 #      ;;
+    "--with-readline")
```

Comment:
Anyways, this is not a big deal. Bundling it probably eliminates more headaches than it causes for most configurations. Windows is the exception here because the embedded copy does not compile under Mingw, but the Msys2 system helpfully includes a copy with the necessary platform-specific patches.